### ISSUE TITLE
Integer parameters can't have negative defaults: "node lacks field: intVal"

### ISSUE THREAD

**Issue by malinges** [onelivesleft/simple_parseopt#3]
First of all, thanks for writing this library!

I'm trying to create an int parameter which defaults to -1, but can't seem to figure out how. The following code:

```nim
import simple_parseopt

let options = get_options:
  myint = -1

echo options.repr
```

doesn't compile:

```
[...]/.nimble/pkgs/simple_parseopt-1.1.0/simple_parseopt.nim(789, 45) get_options_and_supplied
[...]/.nimble/pkgs/simple_parseopt-1.1.0/simple_parseopt.nim(387, 118) param_from_nodes
[...]/simple_parseopt_negint.nim(3, 15) template/generic instantiation of `get_options` from here
[...]/.nimble/pkgs/simple_parseopt-1.1.0/simple_parseopt.nim(1627, 33) template/generic instantiation of `get_options_and_supplied` from here
[...]/.nimble/pkgs/simple_parseopt-1.1.0/simple_parseopt.nim(387, 118) Error: node lacks field: intVal
```

I tried everything I could think of, including explicitly typing the parameter with a signed integer type, or wrapping the literal with parentheses, but to no avail: compilation always fails with the same error.

Am I missing something?

Thanks!

**Comment by onelivesleft** [comment]
Should be fixed in v1.1.1

**Comment by malinges** [comment]
Can confirm, thanks!
